File: spring-boot-oauth2-auth-server/src/main/java/com/bittokazi/oauth2/auth/server/app/models/master/TenantInfo.java
package com.bittokazi.oauth2.auth.server.app.models.master;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

@Setter
@Getter
@Builder
public class TenantInfo implements Serializable {
    private Boolean cpanel = true;
    private Boolean enabledConfigPanel = true;
    private String name = "";
}

File: spring-boot-oauth2-auth-server/src/main/kotlin/com/bittokazi/oauth2/auth/server/app/models/master/TenantInfo.kt
package com.bittokazi.oauth2.auth.server.app.models.master

import com.bittokazi.oauth2.auth.server.config.AppConfig
import java.io.Serializable

data class TenantInfo(
    var cpanel: Boolean = true,
    var enabledConfigPanel: Boolean = true,
    var name: String = "",
    var systemVersion: String = AppConfig.VERSION
) : Serializable
